maths/sum_of_geometric_progression.py

```diff
@@ -0,0 +1,19 @@
+def sum_of_gp(first_term, common_ratio, num_of_terms):
+    """"
+    Find the sum of n terms in a geometric progression.
+    >>> sum_of_gp(1, 2, 10)
+    1023.0
+    >>> sum_of_gp(1, 10, 5)
+    11111.0
+    """
+    if common_ratio == 1:
+        sum = num_of_terms * first_term

```

Comment:
[sum()](https://docs.python.org/3/library/functions.html#sum) is a Python builtin so it should not be used as a variable name.

```suggestion
        return num_of_terms * first_term
```